internal-server.md

```diff
@@ -160,7 +159,8 @@ hello zamboni, this is git@data running gitolite3 3.6.11-2 (Debian) on git 2.27.
 Once you have access you can:
 
 ```
-$ git annex sync --content
+$ git annex copy --to origin
```

Comment:
This is incorrect. `git push` knows what remote to push to by default; but git-annex doesn't. A more fiable command is


```suggestion
$ git annex copy --to="$(git config branch."$(git rev-parse --abbrev-ref HEAD)".remote)"
```

but that's a ton to type.

We could suggest people set up a git alias, but that's also awkward.

I don't have good feelings about this.